gandalf.js

```diff
@@ -1,134 +1,134 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?